src/services/dispatchers/dispatchers/types.ts

```diff
@@ -19,6 +20,30 @@ export interface DispatchersService {
    */
   find(options?: OptionsWithRql): Promise<PagedResult<Dispatcher>>;
 
+  /**
```

Comment:
Double check the jsdocs 😄  Did not find good examples for these 😬

Reply:
Regarding the comments on the interfaces for the Dispatcher findAll / findAll iterator:
https://github.com/ExtraHorizon/javascript-sdk/blob/358ec06cb9aad18774d7dfac1f607c6d399fa948/src/services/auth/oidc/providers/types.ts